pkg/git/verify.go

```diff
@@ -16,13 +16,89 @@
 package git
 
 import (
+	"bytes"
+	"context"
 	"crypto/x509"
 	"encoding/pem"
+	"errors"
 	"fmt"
 
 	cms "github.com/github/smimesign/ietf-cms"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/sigstore/gitsign/pkg/rekor"
+	"github.com/sigstore/rekor/pkg/generated/models"
+	"github.com/sigstore/sigstore/pkg/fulcioroots"
 )
 
+type VerificationSummary struct {
```

Comment:
for the newly exposed types, let's add comments so there's docs in pkg.go.dev.

Reply:
updated, feel free to offer some wording suggestions, thanks!